proto/cacheclient.proto

```diff
@@ -54,33 +54,36 @@ message _SetResponse {
 
 message _DictionaryGetRequest {
   bytes dictionary_name = 1;
-  bytes dictionary_key = 2;
+  repeated bytes dictionary_keys = 2;
```

Comment:
re-using the same numeric slot in the proto with a different type breaks backward compat.

In this case I think it's the right thing to do, but just pointing it out for future consideration when we are considering changes to non-incubating APIs.

(If we have a canary that is testing this stuff I would imagine it will flutter a bit during rollout.)

Reply:
Understood. I was aggressive here since this wasn't hooked up to the backend yet.

Reply:
cool yep i think that's the right call for now.

Reply:
Thank you for calling this out Chris. I agree this is the right thing to do since it is not hooked up yet.